src/utils/standalone_executor.py

```
@@ -0,0 +1,679 @@
+
+    for data in (set_string_variables or []):
+        if '=' not in data:
+            raise ValueError(
+                f'--set-string value "{data}" is incorrectly formatted (expected key=value)')
+        key, raw_value = data.split('=', 1)
+        template_data[key] = raw_value
+
+    placeholder_map: Dict[str, str] = {}
+    for match in _OSMO_TOKEN_PATTERN.finditer(file_text):
+        field = match.group(1).strip()
+        hash_key = 'hash' + str(int(hashlib.md5(field.encode('utf-8')).hexdigest(), 16))
```

Ruff (0.15.9)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)